### PR TITLE
mgr/vol : shortening the name of helper method

### PR DESCRIPTION
**Pull request by neesingh-rh**
Introduced by: https://github.com/ceph/ceph/pull/55838#discussion_r1573655512 and https://github.com/ceph/ceph/pull/55838#issuecomment-2178673729

Fixes: https://tracker.ceph.com/issues/66815
Signed-off-by: Neeraj Pratap Singh <neesingh@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by joscollin** [comment]
Jenkins retest this please

File: src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py [hunk 1]
```diff
@@ -774,7 +774,7 @@ def get_pending_clones(self, snapname):
                     raise VolumeException(-e.args[0], e.args[1])
                 else:
                     try:
-                        # If clone is completed between 'list_all_keys_with_specified_values_from_section'
+                        # If clone is completed between 'filter_keys'
                         # and readlink(track_id_path) call then readlink will fail with error ENOENT (2)
                         # Hence we double check whether track_id is exist in .meta file or not.
                         # Edge case scenario.
```
Comment (rishabh-d-dave) [inline src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py:780]: nit: we don't following coding style so closely but ideally we need  to re-wrap this comment.

**Comment by neesingh-rh** [comment]
> LGTM. IMO we should have a tracker ticket for this and backport it, so that future backports won't have to search for the commit for this change and then backport it during conflicts.

I have created a tracker and added it in the PR description.

**Comment by neesingh-rh** [comment]
jenkins test api

**Review by vshankar — APPROVED** [review]

**Comment by neesingh-rh** [comment]
@vshankar @rishabh-d-dave I think we can proceed with this PR. There's just a small change.

**Comment by vshankar** [comment]
> @vshankar @rishabh-d-dave I think we can proceed with this PR. There's just a small change.

I was likely waiting for jenkins tests to pass and then this got sidelined. Thx for the nudge.

**Pull request merged**